### PR TITLE
AT_07.04.04 | Build History > Verify user can edit DisplayName by clicking on "Edit Build Information" inside the build

### PR DESCRIPTION
**Pull request by asyniagina**
https://trello.com/c/cvnphD1j/2262-at070404-build-history-verify-user-can-edit-displayname-by-clicking-on-edit-build-information-inside-the-build

### CODE REVIEW

File: cypress/e2e/tests/buildHistory.cy.js [hunk 1]
```diff
@@ -213,4 +213,25 @@ describe('buildHistory', () => {
...
+    it('AT_07.04.04 | Build History > Verify user can edit DisplayName by clicking on "Edit Build Information" inside the build', () => {
+        cy.createFreestyleProject(newItemPageData.freestyleProjectName);
+
+        homePage
+            .clickOnScheduleBuildBtn()
+            .clickBuildHistoryLink()
+            .clickBuildLink()
+            .clickEditBuildInfoSideMenuLink()
+            .typeDisplayName(editBuildInformationPageData.displayName)
+            .clickEditBuildInformationSaveBtn()
```
Comment (IrynaKolh) [inline cypress/e2e/tests/buildHistory.cy.js:226]: а чем отличается BuildPage от BuildStatusPage? как по мне при клике на кнопку сохранить открывается BuildPage

File: cypress/e2e/tests/buildHistory.cy.js [hunk 2]
```diff
@@ -213,4 +213,25 @@ describe('buildHistory', () => {
...
+        headerandFooter
+            .clickJenkinsHomeLink()
+            .clickBuildHistoryLink()
+            .clickBuildLink()
+            .clickEditBuildInfoSideMenuLink()
+            .typeDisplayName(editBuildInformationPageData.newDisplayName)
+            .clickEditBuildInformationSaveBtn()
+            .getBuildName()
+            .should('contain', editBuildInformationPageData.newDisplayName);
+    });
```
Comment (S-Lanchik) [inline cypress/e2e/tests/buildHistory.cy.js:236]: I cannot understand why the steps 227-233 are needed. You can put assertion after the step on  the line 226

Comment (verafes) [reply cypress/e2e/tests/buildHistory.cy.js:236]: Everything is OK, in line 226 - a name was created, in line 232 - the name was edited with a new name, and line 235 verifies that the new name is indeed displayed after the replacement

**Review by verafes — APPROVED** [review]

File: cypress/e2e/tests/buildHistory.cy.js (hunk 1 above)
Comment (asyniagina) [reply cypress/e2e/tests/buildHistory.cy.js:226]: BuildPage is a Page inside any Build, BuildHistoryPage is a Page for BuildHistory when you click BuildHistory on the left side menu and see all builds

File: cypress/e2e/tests/buildHistory.cy.js (hunk 2 above)
Comment (S-Lanchik) [reply cypress/e2e/tests/buildHistory.cy.js:236]: > Everything is OK, in line 226 - a name was created, in line 232 - the name was edited with a new name, and line 235 verifies that the new name is indeed displayed after the replacement

I see but when we create build it comes with default name. So, we can check if we can rename it with our name. I think renaming a build two times is not necessary for checking this functionality.

File: cypress/e2e/tests/buildHistory.cy.js (hunk 1 above)
Comment (IrynaKolh) [reply cypress/e2e/tests/buildHistory.cy.js:226]: я спросила в чем разница между BuildPage и BuildStatusPage, а вы мне описали BuildPage и BuildHistoryPage.
при клике на кнопку сохранить открывается BuildPage но никак не BuildStatusPage и у более не BuildHistoryPage

Comment (asyniagina) [reply cypress/e2e/tests/buildHistory.cy.js:226]: @IrynaKolh  Sorry for the misunderstanding, BuildPage and BuildStatusPage are on the same page, they are duplicates. I fixed it and removed BuildStatusPage.

**Review by IrynaKolh — APPROVED** [review]

**Pull request merged**